script/generate_sql

```diff
@@ -66,6 +69,12 @@ def main():
                 with open(os.path.join(basename, filename), "a+") as output_file:
                     output_file.write(full_text)
 
+            if METADATA_FILE in files:
+                shutil.copyfile(
+                    os.path.join(root, METADATA_FILE),
+                    os.path.join(basename, METADATA_FILE),
+                )
```

Comment:
I think we shouldn't copy these to `sql/`, and instead just run `script/publish_metadata` against `templates/`